spanner/value.go

```diff
@@ -2986,7 +3003,7 @@ func decodeStructArray(ty *sppb.StructType, pb *proto3.ListValue, ptr interface{
 			return errDecodeArrayElement(i, pv, "STRUCT", err)
 		}
 		// Decode proto3.ListValue l into struct referenced by s.Interface().
-		if err = decodeStruct(ty, l, s.Interface()); err != nil {
+		if err = decodeStruct(ty, l, s.Interface(), false); err != nil {
```

Comment:
Shouldn't this be:
```suggestion
		if err = decodeStruct(ty, l, s.Interface(), lenient); err != nil {
```

Reply:
No, this is called from decodeStructArray and we don't want to change that function.

Reply:
Ah, yeah, good point. I missed that.